pkg/mock/binding__failpoint_binding__.go

```diff
@@ -0,0 +1,14 @@
+package mock
+
```

Comment:
I think this file should not include

Reply:
Do you mean not to submit to GitHub? @zhouqiang-cl

Reply:
yes

Reply:
done